docs/16_dependencies_licenses.md

```diff
@@ -0,0 +1,159 @@
+# Dependencies licenses
+
+> Summary of the proposed change
+
+Combine license document(s) for all components in monorepo in one document.
+
+# References
+
+## CI Integrations
+
+| Name | Source | License |
+| :---: | :---: | :---: |
+| args  | [pub.dev](https://pub.dev/packages/args) | [BSD](https://github.com/dart-lang/args/blob/master/LICENSE) |
+| http | [pub.dev](https://pub.dev/packages/http) | [BSD](https://github.com/dart-lang/http/blob/master/LICENSE) |
+| meta | [pub.dev](https://pub.dev/packages/meta) | [BSD](https://github.com/dart-lang/sdk/blob/master/pkg/meta/LICENSE) |
+| equatable | [pub.dev](https://pub.dev/packages/equatable) | [MIT](https://github.com/felangel/equatable/blob/master/LICENSE) |
+| archive | [pub.dev](https://pub.dev/packages/archive) | [Apache 2.0](https://github.com/brendan-duncan/archive/blob/master/LICENSE) |
+| firedart | [pub.dev](https://pub.dev/packages/firedart) | [Apache 2.0](https://github.com/cachapa/firedart/blob/master/LICENSE) |
+| dev: |  |  |	
+| lint | [pub.dev](https://pub.dev/packages/lint) | [Apache 2.0](https://github.com/passsy/dart-lint/blob/master/LICENSE) |
```

Comment:
I have two suggestions on how to make dev dependencies more clear in the table. Please let me know what you think! 
The first one is to separate them in another similar table under the sub-sub-section named **Dev dependencies**.
```suggestion
### Dev dependencies
| Name | Source | License |
| :---: | :---: | :---: |
| lint | [pub.dev](https://pub.dev/packages/lint) | [Apache 2.0](https://github.com/passsy/dart-lint/blob/master/LICENSE) |
| ... | ... | ... |
```

Another way is to mark such dependencies with a special sign. I suppose the ⚙️ (`:gear:`) is perfect. But to do that we should inform readers about what this sign means. We can add the following paragraph to the text right under the `Table of licenses` section: 

> In the tables below, we use a special sign to differ usual dependencies from [dev dependencies](https://dart.dev/tools/pub/dependencies#dev-dependencies). Thus, the text ":gear: test" means that the `test` dependency is a dev dependency for the component.

So the table looks like:
```suggestion
| :gear: lint | [pub.dev](https://pub.dev/packages/lint) | [Apache 2.0](https://github.com/passsy/dart-lint/blob/master/LICENSE) |
```

Also, I'm wondering can we do something like this `[:gear:](https://dart.dev/tools/pub/dependencies#dev-dependencies)` so if a reader skips a paragraph with an explanation he or she always can know what the gear means. [⚙️](https://dart.dev/tools/pub/dependencies#dev-dependencies) - looks like this works 😄 

cc @illia-romanenko